README.md

```diff
@@ -0,0 +1,210 @@
+<div align="center">
```

Comment:
- [ ] Kindly, include in your instructions the required db commands that your users will need to run your backend app: ``rails db:create, rails db:migrate``. 👍